include/components/MovableCamera.hxx

```diff
@@ -37,9 +37,14 @@
 class MovableCamera : public Urho3D::Camera {
     URHO3D_OBJECT(MovableCamera, Camera);
 
+    ///Defines the moving speed of the camera
     float move_speed = DEFAULT_MOVE_SPEED;
+    ///Defines the mouse sensitivity
     float mouse_sensitivity = DEFAULT_MOVE_SENSITIVITY;
 
+    /**
+    * Maps the keys so that you are able to move the cameara with keys w,s,a,d
+    **/
```

Comment:
please reword this phrase, say something like "Maps the keys to enable the camera movement with w,s, a,d". besides "Cameare -> Camera"